sass/color/_functions.scss

```diff
@@ -337,12 +348,3 @@ $_enhanced-accessibility: false;
 @function chart-brushes() {
     @return if($_enhanced-accessibility, charts.$brushes-color-blind, charts.$brushes-regular);
 }
-
-/// Returns a list of H, S, L values for a given color.
-/// @access public
-/// @param {Color} $color - The color to generate HSL values for.
-/// @group Palettes
-/// @returns { List } - The list of values.
-@function to-hsl($color) {
-    @return (color.hue($color), color.saturation($color), color.lightness($color));
-}
```

Comment:
We shouldn't remove this.

Reply:
This function was duplicated so I just removed the second one. See lines 285 - 294

Reply:
Sorry, didn't see it.